### DISCUSSION TITLE
Add location automatically

### DISCUSSION THREAD

**Discussion by nexus256** [joleys/niko-home-control-II#142] [Ideas]
Can we add the location (Area) automatically for each device during the setup?  I think this information is also present in the NHC mqtt information.
Same like the Philips Hue integration: if a device is added with this integration it will be assigned to the correct Area in Home Assistant.  If the Area does not exist, it will be created first. (in Hue it is called 'Room') (see https://github.com/home-assistant/core/blob/dev/homeassistant/components/hue/v2/device.py?)

**Comment by tijsverkoyen** [comment]
I have added support for the suggested_area in https://github.com/joleys/niko-home-control-II/pull/143
